### PR TITLE
UIE-204 Narrow Ajax Usage pt3

### PR DESCRIPTION
**Pull request by msilva-broad**
- switch first wave of shared src/workspaces code to call Ajax sub-modules directly.
- updated related unit test mocks.
- updated workflows WorkflowView tests to honor Ajax shifts in useWorkspace related changes, and shifted its own Ajax calls.

### What
-

### Why
- improve code maintainability and potential for code sharing.

### Testing strategy

- [ ]

### CODE REVIEW

File: src/pages/workspaces/workspace/workflows/WorkflowView.js [hunk 1]
```diff
@@ -507,7 +511,7 @@ export const WorkflowView = _.flow(
                   } = modifiedConfig;
                   // will only match if the current root entity type comes from a snapshot
                   const snapshot = _.find({ metadata: { name: modifiedConfig.dataReferenceName } }, availableSnapshots);
-                  Ajax().Metrics.captureEvent(Events.workflowLaunch, {
+                  void Metrics().captureEvent(Events.workflowLaunch, {
```
Comment (cahrens) [inline src/pages/workspaces/workspace/workflows/WorkflowView.js:514]: Does the void make it clear that we don't care about awaiting?

Comment (msilva-broad) [reply src/pages/workspaces/workspace/workflows/WorkflowView.js:514]: correct, and it removes a lint warning for the same reason.

I may have missed a couple of these in a recent PR, but try to fix them as I see them.

Comment (cahrens) [reply src/pages/workspaces/workspace/workflows/WorkflowView.js:514]: Cool! It would be good to do it consistently then.

Comment (msilva-broad) [reply src/pages/workspaces/workspace/workflows/WorkflowView.js:514]: ok yeah, I see a few I missed in other PR files.... I'll fix um now

**Review by cahrens — APPROVED** [review]

**Review by sam-schu — APPROVED** [review]
Workflows changes look good!

**Pull request merged**